### PR TITLE
Genomic Browser

### PR DESCRIPTION
**Pull request by christinerogers**
new Genomic Browser module  for viewing and searching CNV and SNP datasets.
Originally developed for NeuroDevNet
Includes schema changes and patch to add module-specific permission and LorisMenu item
No gene data included, only table schemas.

### CODE REVIEW

File: SQL/2014-12-12-GenomicBrowser.sql [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+-- Add menu permissions
+INSERT INTO LorisMenuPermissions (MenuID, PermID) SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='genomic_browser_view_site' AND m.Label='Genomic Browser';
+INSERT INTO LorisMenuPermissions (MenuID, PermID) SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='genomic_browser_view_allsites' AND m.Label='Genomic Browser';
+
+-- Add tables
+--
+-- Table structure for table `gene`
+--
+DROP TABLE IF EXISTS `gene`;
```
Comment (driusan) [inline SQL/2014-12-12-GenomicBrowser.sql]: Patches shouldn't drop tables

File: SQL/2015-01-07-GenomicBrowser.sql [hunk 2]
```diff
@@ -0,0 +1,110 @@
+-- Add permissions
```
Comment (driusan) [inline SQL/2015-01-07-GenomicBrowser.sql]: This file seems to be a duplicate with a different name?

File: modules/genomic_browser/php/NDB_Menu_Filter_genomic_browser.class.inc [hunk 3]
```diff
@@ -0,0 +1,381 @@
+<?php
```
Comment (driusan) [inline modules/genomic_browser/php/NDB_Menu_Filter_genomic_browser.class.inc]: This file seems to have PHPCS run on it, so it should be added to Travis. Something like:

"- vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/genomic_browser"

(to automatically add all PHP from the module.)

File: .travis.yml [hunk 4]
```diff
@@ -94,3 +94,4 @@ script:
     # Run PHPCS on specific modules
     - vendor/bin/phpcs --standard=docs/LorisCS.xml modules/mri_upload/php/NDB_Menu_Filter_mri_upload.class.inc
     - vendor/bin/phpcs --standard=docs/LorisCS.xml modules/mri_upload/php/File_Decompress.class.inc
+    - vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/genomic_browser
```
Comment (christinerogers) [inline .travis.yml]: @driusan any changes to make here?

**Comment by driusan** [comment]
There's a couple problems according to the code review checklist, but since this pull request predates the checklist existing and it's up to the standards of when it was written, I'm tagging it as passed. I think it should be merged and they can be fixed after.

The main issues are:
1. The function doc comments aren't very meaningful (some of them just say the function name. They don't say what they do or why they're overridden.)
2. There are no tests.

**Pull request merged**